### PR TITLE
OnlineMediaTechnologiesLtd.AVSDiscCreator version 6.2.4.564

### PR DESCRIPTION
**Pull request by techflashYT**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/15940)

### CODE REVIEW

**Comment by sanikdah** [comment]
It's approved so why isn't it merged?

**Comment by denelon** [comment]
@sanikdah one of our moderators needs to review to apply the "Moderator-Approved" label.

**Comment by sanikdah** [comment]
Ahh ok I see, extra measures past the azure pipelines testing, got it.

**Comment by denelon** [comment]
OnlineMediaTechnologiesLtd seems quite long. I would remove the "Ltd" from the end of the PackageIdentifier and subsequently the directory name.

**Comment by denelon** [comment]
The short description shouldn't have "setup" in the name unless the product is actually used to setup a disc.

From: https://onlinehelp.avs4you.com/avs-disc-creator/index.aspx

"AVS Disc Creator is a compact and fully functional application that lets you perform different burning tasks with any kind of files."

**Comment by sanikdah** [comment]
> The short description shouldn't have "setup" in the name unless the product is actually used to setup a disc.
> 
> From: https://onlinehelp.avs4you.com/avs-disc-creator/index.aspx
> 
> "AVS Disc Creator is a compact and fully functional application that lets you perform different burning tasks with any kind of files."

I would have to remake the entire PR then because last time I tried to change the files it just gave me a manifest content error.

**Comment by KaranKad** [comment]
If you know how to use git, you can rename the files. Otherwise you can create a new pr as wingetcreate does the most for you.

File: manifests/o/OnlineMediaTechnologiesLtd/AVSDiscCreator/6.2.4.564/OnlineMediaTechnologiesLtd.AVSDiscCreator.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: OnlineMediaTechnologiesLtd.AVSDiscCreator
+PackageVersion: 6.2.4.564
+PackageLocale: en-US
+Publisher: Online Media Technologies Ltd.
+PackageName: AVS Disc Creator
+License: Copyright AVS4YOU all rights reserved.
+ShortDescription: AVS Disc Creator Setup
```
Comment (denelon) [inline manifests/o/OnlineMediaTechnologiesLtd/AVSDiscCreator/6.2.4.564/OnlineMediaTechnologiesLtd.AVSDiscCreator.locale.en-US.yaml:9]:
```suggestion
ShortDescription: AVS Disc Creator is a compact and fully functional application that lets you perform different burning tasks with any kind of files.
```

**Review by denelon — CHANGES_REQUESTED** [review]

**Comment by sanikdah** [comment]
There you go.

**Review by denelon — APPROVED** [review]

**Pull request merged**